lib/index.js

```diff
@@ -120,21 +131,42 @@ export default class HyperList {
       throw new Error('Missing required `generate` function');
     }
 
-    if (Number(config.total) !== Number(config.total)) {
+    if (!isNumber(config.total)) {
       throw new Error('Invalid required `total` value, expected number');
     }
 
+    if (!Array.isArray(config.itemHeight) && !isNumber(config.itemHeight)) {
+      throw new Error(`
+        Invalid required \`itemHeight\` value, expected number or array
+      `.trim());
+    }
+    else if (isNumber(config.itemHeight)) {
+      this[_itemHeights] = Array(config.total).fill(config.itemHeight);
+    }
+    else {
+      this[_itemHeights] = config.itemHeight;
+    }
+
+    // Reuse the item positions if refreshed, otherwise set to empty array.
+    this[_itemPositions] = this[_itemPositions] || [];
```

Comment:
Hmm yes, my argument is false thanks. Though, any truthy value would be taken and `=== undefined` is cleaner imo (and faster).